score.py

```diff
@@ -0,0 +1,214 @@
+from collections import Counter, namedtuple
+
+from nameinternal import get_card_metadata, get_score_bonus
+from save import Savefile
+
+class Score:
+    def __init__(self, name: str, count: int, score: int = 0, should_show: bool = False):
+        self.score_bonus = score
+        self.should_show = should_show
+        if name:
+            data = get_score_bonus(name)
+            if data:
+                self._name = data.name
+                self._description = data.description
+                self._format_string = data.format_string.format(count = count)
```

Comment:
There should be a `self._format_string` definition somewhere for if `name` is not provided, so that the attribute exists.